Automattic/pocket-casts-ios
# Fix Xcode file that's out of place

## To test

1. Open Xcode
2. Verify StorageManager is in the Utilities group

## Checklist
